dapr/proto/placement/v1/placement.proto

```diff
@@ -46,4 +46,5 @@ message Host {
   int64 load = 3;
   repeated string entities = 4;
   string id = 5;
+  string pod = 6;
```

Comment:
Is this optional? Can this be k8s agnostic, maybe hostInfo?

Reply:
Yes, this is an optional field and will be filled only in k8s mode.
Since this will be empty in self-hosted mode, maybe it makes sense to leave the `pod` as a name making it a bit more obvious that it is only expected in k8s? WDYT?

Reply:
OK.